Author: Jasper Grimm <jasper.grimm@york.ac.uk>
use external Sox library instead of fetching and building at build time
--- CMakeLists.txt.orig	2024-04-10 22:50:03.376931244 +0000
+++ CMakeLists.txt	2024-04-10 22:51:00.861834095 +0000
@@ -167,7 +167,7 @@
 
 add_subdirectory(torchaudio/csrc)
 if (BUILD_SOX)
-  add_subdirectory(third_party/sox)
+  # add_subdirectory(third_party/sox)
   add_subdirectory(torchaudio/csrc/sox)
 endif()
 if (USE_FFMPEG)
